components/avatar.js

```diff
@@ -0,0 +1,12 @@
+export default function Avatar({ name, picture }) {
+  return (
+    <div className="flex items-center">
+      <img
```

Comment:
Should use next/image: https://nextjs.org/docs/basic-features/image-optimization